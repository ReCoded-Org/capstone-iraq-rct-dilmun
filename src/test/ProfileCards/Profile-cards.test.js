import React from 'react'
import renderer from 'react-test-renderer'
import { ToastProvider } from 'react-toast-notifications'
import { Provider } from 'react-redux'
import { BrowserRouter as Router } from 'react-router-dom'
import store from '../../redux/store'
import ProfileCards, { Header } from '../../components/ProfileContentCard'

it('Test: Profile Cards', () => {
  const tree = renderer
    .create(
      <Router>
        <ToastProvider autoDismiss autoDismissTimeout="2500">
          <Provider store={store}>
            <Header />
            <ProfileCards />
          </Provider>
        </ToastProvider>
      </Router>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
