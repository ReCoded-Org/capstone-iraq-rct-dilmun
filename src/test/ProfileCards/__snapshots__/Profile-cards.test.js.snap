// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test: Profile Cards 1`] = `
Array [
  <div
    className=" grid  md:px-20 md:mx-20 p-5"
  >
    <div
      className="grid grid-cols-2 bg-white mx-3 px-4 py-2 rounded-2xl shadow-lg"
    >
      <div
        className="font-bold text-xl self-center"
      >
        profile.myItems
      </div>
      <div
        className="justify-self-end"
      >
        <button
          className="bg-blue  text-md   text-white py-2 text-center  space-x-1 items-center   w-full shadow-md hover:shadow-none rounded-full transition duration-300 ease-in focus:outline-none hover:bg-darkBlue px-3  "
          onClick={[Function]}
          type="button"
        >
          <span
            className=""
          >
             
            home.addItem
             
          </span>
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-plus fa-w-14 "
            data-icon="plus"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M416 208H272V64c0-17.67-14.33-32-32-32h-32c-17.67 0-32 14.33-32 32v144H32c-17.67 0-32 14.33-32 32v32c0 17.67 14.33 32 32 32h144v144c0 17.67 14.33 32 32 32h32c17.67 0 32-14.33 32-32V304h144c17.67 0 32-14.33 32-32v-32c0-17.67-14.33-32-32-32z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </button>
      </div>
    </div>
  </div>,
  <div
    className=" grid  md:px-20 md:mx-20 px-5"
  >
    <div
      className="grid grid-cols-1 md:grid-cols-4   m-3 px-4 py-2 rounded-2xl border  shadow-inner bg-white"
    >
      <div
        className="grid gap-2 max-h-40 overflow-auto col-span-2"
      >
        <div
          className="font-bold text-xl capitalize"
        >
           
          No Content
        </div>
        <p
          className=""
        >
          
        </p>
        <hr
          className="md:hidden"
        />
      </div>
      <div
        className="justify-self-start md:justify-self-center grid gap-2 text-blue mt-2 md:mt-0    "
      >
        <div>
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-clock fa-w-16 "
            data-icon="clock"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M256,8C119,8,8,119,8,256S119,504,256,504,504,393,504,256,393,8,256,8Zm92.49,313h0l-20,25a16,16,0,0,1-22.49,2.5h0l-67-49.72a40,40,0,0,1-15-31.23V112a16,16,0,0,1,16-16h32a16,16,0,0,1,16,16V256l58,42.5A16,16,0,0,1,348.49,321Z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
           
          0W 0D 0h ago
        </div>
        <div>
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-eye fa-w-18 mx-1"
            data-icon="eye"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 576 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M572.52 241.4C518.29 135.59 410.93 64 288 64S57.68 135.64 3.48 241.41a32.35 32.35 0 0 0 0 29.19C57.71 376.41 165.07 448 288 448s230.32-71.64 284.52-177.41a32.35 32.35 0 0 0 0-29.19zM288 400a144 144 0 1 1 144-144 143.93 143.93 0 0 1-144 144zm0-240a95.31 95.31 0 0 0-25.31 3.79 47.85 47.85 0 0 1-66.9 66.9A95.78 95.78 0 1 0 288 160z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
          0
        </div>
      </div>
      <div
        className="justify-self-end self-center "
      >
        <button
          className="bg-yellow px-4 py-1 rounded-full text-pureWhite hover:shadow-none focus:outline-none shadow-md  transition duration-300 ease-in-out "
          onClick={[Function]}
          type="button"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-binoculars fa-w-16 "
            data-icon="binoculars"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M416 48c0-8.84-7.16-16-16-16h-64c-8.84 0-16 7.16-16 16v48h96V48zM63.91 159.99C61.4 253.84 3.46 274.22 0 404v44c0 17.67 14.33 32 32 32h96c17.67 0 32-14.33 32-32V288h32V128H95.84c-17.63 0-31.45 14.37-31.93 31.99zm384.18 0c-.48-17.62-14.3-31.99-31.93-31.99H320v160h32v160c0 17.67 14.33 32 32 32h96c17.67 0 32-14.33 32-32v-44c-3.46-129.78-61.4-150.16-63.91-244.01zM176 32h-64c-8.84 0-16 7.16-16 16v48h96V48c0-8.84-7.16-16-16-16zm48 256h64V128h-64v160z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </button>
        <button
          className="bg-red mx-2 px-4 py-1 rounded-full text-white hover:shadow-none focus:outline-none shadow-md  transition duration-300 ease-in-out "
          onClick={[Function]}
          type="button"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-trash-alt fa-w-14 "
            data-icon="trash-alt"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M32 464a48 48 0 0 0 48 48h288a48 48 0 0 0 48-48V128H32zm272-256a16 16 0 0 1 32 0v224a16 16 0 0 1-32 0zm-96 0a16 16 0 0 1 32 0v224a16 16 0 0 1-32 0zm-96 0a16 16 0 0 1 32 0v224a16 16 0 0 1-32 0zM432 32H312l-9.4-18.7A24 24 0 0 0 281.1 0H166.8a23.72 23.72 0 0 0-21.4 13.3L136 32H16A16 16 0 0 0 0 48v32a16 16 0 0 0 16 16h416a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </button>
      </div>
    </div>
  </div>,
]
`;
